packages/blockchain-tools/src/interfaces/Web3Server.ts

```diff
@@ -40,12 +47,12 @@ export default class Web3Server extends ChainBaseClass {
     }
     const acct = this.web3.eth.accounts.privateKeyToAccount(walletSeed)
     const minter = acct.address
-    const instance = new this.web3.eth.Contract(Abi721, contractId)
+    const instance = new this.web3.eth.Contract(Abi721inc, contractId)
```

Comment:
Do we still use the non-auto-incrementing 721? If not, we could get rid of it.

Reply:
Yes, for TBAs we pass the entity ID as tokenid